app/src/main/java/de/westnordost/streetcomplete/map/PinIcons.kt

```diff
@@ -59,5 +60,20 @@ import de.westnordost.streetcomplete.R
     when (map["tourism"]) {
         "information" -> return R.drawable.ic_pin_information
     }
+    if ((map.containsKey("addr:housenumber") || map.containsKey("addr:housename"))
+        && !map.containsAnyKey("amenity", "craft", "leisure", "office", "shop", "tourism")){
```

Comment:
Off-putting, for what?  I didn't really get anyway why housenumbers of shops should not be shown. So what if there are multiple housenumbers being displayed with the same number (because they are in the same building).

So anyway, I agree that this could be dropped.

Reply:
Without this check, in e.g. opening hours quest shops with house numbers look different to those without.
Shops without would have a pink dot, shops with wouldn't.
So actually I am in favour of keeping it, and I could add a comment to the code to that effect?